usaspending_api/references/models.py

```diff
@@ -521,16 +521,13 @@ class CFDAProgram(DataSourceTrackedModel):
     recovery = models.TextField(blank=True, null=True)
     omb_agency_code = models.TextField(blank=True, null=True)
     omb_bureau_code = models.TextField(blank=True, null=True)
-    # published_date = models.DateTimeField(blank=True, null=True)
-    # archived_date = models.DateTimeField(blank=True, null=True)
     published_date = models.TextField(blank=True, null=True)
     archived_date = models.TextField(blank=True, null=True)
     create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     update_date = models.DateTimeField(auto_now=True, null=True)
 
     class Meta:
```

Comment:
Since we're recreating the table, what do you think about adding a unique index on `program_number`? At least on my local db, that field is already unique, but could be a nice line of defense for future updates this info.